release-notes-1.1.1.md

```diff
@@ -1,26 +1,42 @@
-# Overview to Ballerina 1.1.1
+# Overview of Ballerina 1.1.1
+
+Ballerina 1.1.1 is a patch release iteration of its previous 1.1.0 release, which introduces the below functionalities. Also, this release addresses a few [bug fixes](https://github.com/ballerina-platform/ballerina-lang/issues?utf8=✓&q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+1.1.1%22+is%3Aclosed+)
+and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?utf8=✓&q=is%3Aissue+milestone%3A%22Ballerina+1.1.1%22+is%3Aclosed+label%3AType%2FImprovement+). 
+
+## Ballerina Update Tool
+
+The inbuilt updating capability (i.e., the Ballerina Update tool) introduced in Ballerina 1.1.0 is improved with the following usages. Now, you can update Ballerina by executing the below commands based on the Ballerina and JBallerina versions that you have installed.
+
+<table border=1>
+<tr>
```

Comment:
Also, shall we flip the table and change usage to something like "Installation and Active version"

Reply:
Done.